lib/utils/data/globals.dart

```diff
@@ -78,20 +79,27 @@ double verticalPadding(BuildContext context, double size) =>
 
 List<String> timeZones = [];
 
-List<String> wallpapers = [
-  "images/wallpapers/dahliaOS_white_logo_pattern_wallpaper.png",
-  "images/wallpapers/dahliaOS_white_wallpaper.png",
-  "images/wallpapers/Gradient_logo_wallpaper.png",
-  "images/wallpapers/Three_Bubbles.png",
-  "images/wallpapers/Bubbles_wallpaper.png",
-  "images/wallpapers/Mountains_wallpaper.png",
-  "images/wallpapers/mountain.jpg",
-  "images/wallpapers/forest.jpg",
-  "images/wallpapers/modern.png",
-  "images/wallpapers/modern_dark.png",
-  "images/wallpapers/wood.jpg",
-  "images/wallpapers/beach.jpg",
-];
+Future<List<Wallpaper?>?> getWallpapers() async {
```

Comment:
instead of returning a List<Wallpaper?>? return a List<Wallpaper>? as we don't need the nullability (will make code that depend on this function easier to work with)